netbox_librenms_plugin/templates/netbox_librenms_plugin/_ipaddress_sync_content.html

```
@@ -131,3 +144,58 @@
+                    <th>Interface</th>
+                    <th class="text-end">Migrate</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for ip in ip_sync.movable_ips %}
+                <tr>
+                    <td>{{ ip.address }}</td>
+                    <td>{{ ip.interface_name|default:"-" }}</td>
+                    <td class="text-end">
+                        {% if has_write_permission and migrated_to_winner %}
+                        <button type="button"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)